content/updates/quality/1-The-quality-score.md

```diff
@@ -0,0 +1,32 @@
+# Introducing our Quality score!
+
+## A new way of scoring...
+::: meta-data type="New" ee-only="false" link-to-doc="../articles/understand-data-quality.html#about-data-quality-insights"
+:::
+Measuring your product quality is now easier than ever thanks to the **quality score**, our new simplified way to score your product. It replaces the enrichment and consistency grades for better clarity.
+
+![The new quality score](../img/quality-score.png)
+
+You can now find this score next to the completeness in the header of each product page, as well as in the product grid. Don't hesitate to use it as a filter to focus your attention on the products with low-quality scoring.
+
+![The new quality score in the product edit form](../img/quality-score-in-pef.png)
+
+## ...All your products...
```

Comment:
I don't understand the ... in every title

Reply:
The three titles form a single title. "A new way of scoring / All your products / On selected attributes!"

Reply:
ah! ok :-)